rendermodules/lens_correction/schemas.py

```diff
@@ -154,6 +154,9 @@ class LensCorrectionParameters(ArgSchema):
                      "unspecified or create temporary directory if None)"))
     SIFT_params = Nested(SIFTParameters)
     align_params = Nested(AlignmentParameters)
+    max_threads_SIFT = Int(
+        required=False, default=1, description=(
```

Comment:
is there a reason to have this as 1 as a default?

Reply:
Not really -- I prefer to have it low, but if you think it's useful to to have it default to more we can do that.

Reply:
How about 3?